client/consensus/babe/src/lib.rs

```diff
@@ -894,7 +894,7 @@ where
 		Box::pin(
 			self.env
 				.init(block)
-				.map_err(|e| sp_consensus::Error::ClientImport(format!("{:?}", e))),
+				.map_err(|e| ConsensusError::ClientImport(format!("{:?}", e))),
```

Comment:
```suggestion
				.map_err(|e| ConsensusError::ClientImport(e.to_string())),
```

(We want to format errors through `Display` if we can.)